src/ppspp_options.erl

```diff
@@ -173,20 +183,22 @@ unpack( <<?PPSPP_LIVE_DISCARD_WINDOW, Maybe_Discard_Window/binary>>,
                  16#ffffffffffffffff -> ppspp_live_discard_window_keep_all_chunks;
```

Comment:
Line 183 is too long:                  16#ffffffffffffffff -> ppspp_live_discard_window_keep_all_chunks;.